hosted/eks/p1/p1_provisioning_test.go

```diff
@@ -77,7 +80,9 @@ var _ = Describe("P1Provisioning", func() {
 			Eventually(func() bool {
 				cluster, err := ctx.RancherAdminClient.Management.Cluster.ByID(cluster.ID)
 				Expect(err).To(BeNil())
-				return cluster.Transitioning == "error" && strings.Contains(cluster.TransitioningMessage, "is not unique within the cluster")
+				// checking for both the messages since different operator version shows different messages. To be removed once the message is updated.
+				// New Message: NodePool names must be unique within the [c-dnzzk] cluster to avoid duplication
+				return cluster.Transitioning == "error" && (strings.Contains(cluster.TransitioningMessage, "is not unique within the cluster") || strings.Contains(cluster.TransitioningMessage, "NodePool names must be unique"))
```

Comment:
nice, I introduced the change and did not realized that with other operator version the message can be different

Reply:
Some of the messages on nightly builds are different from the ones on released.